src/hooks/useSignInDialog/Steps/Steps.style.ts

```diff
@@ -1,28 +1,41 @@
 import { Text } from '@/shared/components'
+import { SvgGlyphWarning } from '@/shared/icons/GlyphWarning'
 import { sizes, colors } from '@/shared/theme'
 import styled from '@emotion/styled'
 
 type StepWrapperProps = {
   centered?: boolean
+  withBottomBar?: boolean
 }
 
 export const StepWrapper = styled.div<StepWrapperProps>`
   width: 100%;
   text-align: ${({ centered }) => (centered ? 'center' : 'left')};
   display: flex;
+  flex-grow: 1;
   flex-direction: column;
   align-items: ${({ centered }) => (centered ? 'center' : 'flex-start')};
   margin-top: ${({ centered }) => (centered ? sizes(12) : sizes(2))};
 `
 
 export const StepTitle = styled(Text)`
-  line-height: ${sizes(8)};
-  margin-top: ${sizes(4)};
-  max-width: 400px;
+  margin-top: ${sizes(2)};
 `
 
 export const StepSubTitle = styled(Text)`
   margin-top: ${sizes(2)};
-  color: ${colors.gray[200]};
   max-width: 400px;
 `
+
+export const StepFooter = styled.div`
+  margin-top: 24px;
+  background-color: ${colors.gray[800]};
+  width: 100%;
+  outline: 24px solid ${colors.gray[800]};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+export const BottomBarIcon = styled(SvgGlyphWarning)`
+  margin-right: 10px;
```

Comment:
This isn't the correct value from the design. It should be 8/sizes(2). In general, most of the time when you cannot use `sizes`, you may have the wrong value. Take a look at the selected node on the left:

<img width="780" alt="Screenshot 2021-04-16 at 11 30 52" src="https://user-images.githubusercontent.com/12646744/115004607-43411d80-9ea7-11eb-8dc2-45337b2bffb2.png">

<img width="656" alt="Screenshot 2021-04-16 at 11 31 00" src="https://user-images.githubusercontent.com/12646744/115004632-489e6800-9ea7-11eb-9884-9f02e04267cb.png">